src/objects/logo-group/logo-group.scss

```diff
@@ -1,21 +1,41 @@
 @use '../../compiled/tokens/scss/size';
 @use '../../mixins/ms';
 
+/// The Logo Group container is where most of the magic happens. We use flex
+/// instead of grid because we want to support centered justification later (if
+/// we used grid, the bottom row would always be left/start justified).
 .o-logo-group {
   align-items: center;
   display: flex;
   flex-wrap: wrap;
+  gap: size.$spacing-gap-logo-group;
```

Comment:
This actually results in slightly less whitespace than the previous technique since there aren't margins to double up, but my eye prefers having consistent whitespace between _and_ around the items (compared to before, when there was half as much around) and I'm not bothered by the space reduction. But, I'm open to suggestion on this!